packages/manager/modules/pci/src/components/pci-order-command/pci-order-command.controller.js

```diff
@@ -0,0 +1,49 @@
+import isString from 'lodash/isString';
+import isNumber from 'lodash/isNumber';
+import isBoolean from 'lodash/isBoolean';
+import isNull from 'lodash/isNull';
+import isArray from 'lodash/isArray';
+import isPlainObject from 'lodash/isPlainObject';
+import isEmpty from 'lodash/isEmpty';
+import forOwn from 'lodash/forOwn';
+
+export default class {
+  $onInit() {
+    this.orderDataKeys = this.parameterKeys;
+    if (!this.parameterKeys) {
+      this.orderDataKeys = Object.keys(
+        this.flattenObject(this.orderData, '', {}, false),
+      );
+    }
+  }
+
+  getRequestData() {
+    return JSON.stringify(this.orderData, undefined, 4);
+  }
+
+  flattenObject(o, prefix = '', resultMerging = {}, keepNull = true) {
```

Comment:
Nitpick: east to understand if you can rename parameters meaningfully and add method documentation.

Reply:
Done :)